disttask/framework/dispatcher/dispatcher.go

```diff
@@ -244,6 +246,10 @@ func (d *BaseDispatcher) onRunning() error {
 	return nil
 }
 
+func (d *BaseDispatcher) onFinished() error {
+	return d.taskMgr.TransferSubTasks2History(d.task.ID)
```

Comment:
for `show import job` of running jobs, it will query subtask table for import_rows, if the task finished in the middle, this value might be 0.

better keep subtasks rows for some time, then move it into history, then gc

Reply:
why not update the result to task_job table regularly?

Reply:
There can be 2 mechanism:
1. collect rows info regularly, and update them to task table once import finished always
2. `show import job` get the rows info from subtask table firstly. If they do not exist, get them from task table.